src/System.DirectoryServices.AccountManagement/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -1,4 +1,23 @@
-<!-- Temporary workaround to get build to work. -->
-credui.dll!CredUIParseUserNameW
-advapi32.dll!LsaNtStatusToWinError
 activeds.dll!ADsOpenObject
```

Comment:
As mentioned above, I plan to enforce the checks for netstandard libraries by default though. They don't seem onerous.I'll take it off any that violate and we don't care about that, including DirectoryServices.*.